<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_removeProduct_master</class>
 <widget class="QDialog" name="Dialog_removeProduct_master">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>211</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>378</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>378</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_remove_product">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_product_id_for_delete">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Идентификатор товара</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_product_name_for_delete">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>119</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Товар</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Удаление товара&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
